introduction.md

```diff
@@ -0,0 +1 @@
+test sj
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
I would luv to go over javascript path on Code School